code/game/objects/items/storage/backpack.dm

```diff
@@ -352,33 +352,53 @@
 
 /obj/item/storage/backpack/duffelbag/cursed
 	name = "living duffel bag"
-	desc = "A cursed clown duffel bag that hungers for food of any kind. Putting some food for it to eat inside of it should distract it from eating you for a while. A warning label on one of the duffel bag's sides cautions against feeding your \"new pet\" anything poisonous..."
+	desc = "A cursed clown duffel bag that hungers for food of any kind."
 	icon_state = "duffel-curse"
 	inhand_icon_state = "duffel-curse"
-	slowdown = 1.3
+	slowdown = 2
+	item_flags = DROPDEL
 	max_integrity = 100
 	///counts time passed since it ate food
 	var/hunger = 0
 
-/obj/item/storage/backpack/duffelbag/cursed/Initialize()
+/obj/item/storage/backpack/duffelbag/cursed/examine(mob/user)
+	. = ..()
+	. += "<span class='danger'>A warning label suggests that it eats food inside.</span>"
+	. += "<span class='danger'>If that food happens to be a horribly ruined, burned mess the chef scrapped out of the microwave, then it might have negative effects on the bag...</span>"
+	
+	if(hunger > 25)
+		. += "<span class='danger'>The bag is growling for food...</span>"
+
+/obj/item/storage/backpack/duffelbag/cursed/equipped(mob/living/carbon/human/user, slot)
 	. = ..()
 	START_PROCESSING(SSobj,src)
-	ADD_TRAIT(src, TRAIT_NODROP, "duffelbag")
+	ADD_TRAIT(src, TRAIT_NODROP, CURSED_ITEM_TRAIT)
+	ADD_TRAIT(user, TRAIT_CLUMSY, CURSED_ITEM_TRAIT)
+	ADD_TRAIT(user, TRAIT_PACIFISM, CURSED_ITEM_TRAIT)
+	ADD_TRAIT(user, TRAIT_DUFFEL_CURSED, CURSED_ITEM_TRAIT)
+
+/obj/item/storage/backpack/duffelbag/cursed/dropped(mob/living/carbon/human/user)
+	REMOVE_TRAIT(user, TRAIT_DUFFEL_CURSED, CURSED_ITEM_TRAIT)
+	REMOVE_TRAIT(user, TRAIT_CLUMSY, CURSED_ITEM_TRAIT)
+	REMOVE_TRAIT(user, TRAIT_PACIFISM, CURSED_ITEM_TRAIT)
+	STOP_PROCESSING(SSobj,src)
```

Comment:
```suggestion
	STOP_PROCESSING(SSobj,src)
	. = ..()
```

Reply:
You missed a few lines, the return is further down, but sure I get what you're saying.

Reply:
Oh you're right, I did my bad